terraform/main.tf

```diff
@@ -38,7 +38,7 @@ module "aws_deploy-ap-southeast-1" {
   instance_type = "m4.large"
 
   epoch = {
-    version = "0.19.0"
+    package = "https://s3.eu-central-1.amazonaws.com/aeternity-epoch-releases/epoch-latest-ubuntu-x86_64.tar.gz"
```

Comment:
How does guarantee comes that `s3.eu-central-1.amazonaws.com` serves latest version of file? (I mean - eu-central-1 may have stale copy while another asia or so have a recent one.)

Reply:
we out our images only in eu-central-1, aws it self to not synch it cross region

Reply:
we upload image just to eu-central-1 s3 bucket, aws it self do not synch it to other regions